packages/main/src/Avatar.js

```diff
@@ -245,6 +321,31 @@ class Avatar extends UI5Element {
 			},
 		};
 	}
+
+	_onclick(event) {
+		event.isMarked = "avatar";
+		if (this.interactive) {
+			event.preventDefault();
+			// Prevent the native event and fire custom event because otherwise the noConfict event won't be thrown
+			this.fireEvent("click");
+		}
+	}
+
+	_onkeyup(event) {
+		if (this.interactive && !event.shiftKey && (isSpace(event) || isEnter(event))) {
```

Comment:
I think there is common keyboard handling rule that with Space the event should be fired onkeyup, but with Enter onkeydown.